apps/supernova/src/components/silences/RecreateSilence.jsx

```diff
@@ -198,7 +202,13 @@ const RecreateSilence = (props) => {
 
               <Form className="mt-6">
                 <FormRow>
-                  <TextInput required label="Silenced by" value={formState.createdBy} disabled />
+                  <TextInput
+                    required
+                    label="Silenced by"
+                    value={formState.createdBy}
+                    disabled={!isNameEditable}
```

Comment:
the same here. You wouldn't need to check with a local variable if the authData is set correctly with a flag which describes that there is no authentication.